### DISCUSSION TITLE
请教一下大家，使用 elasticsearch-logger 插件传入的日志如何做 ILM？

### DISCUSSION THREAD

**Discussion by LisonFan** [apache/apisix#10852] [Q&A]
网上找了一下，elasticsearch-logger 插件的 field.index 参数好像不支持变量，也就是不支持写入的时候接上日期做索引；写入 es 的索引名称就是固定的，该如何做 ILM？有没有大佬传授下经验

**Comment by snowleopard331** [comment]
目前我也遇到了这个问题，想知道一下答案

**Reply by myhgc** [reply]
这个问题有人解决了嘛？
